### PR TITLE
📝(project) add uploaded file and downloaded file statement templates

### PR DESCRIPTION
**Pull request by wilbrdt**
We need to emit events for downloading/uploading a file under the LMS. These
events should be common to different LMS.

For this, statement templates added to the common profile:
- uploaded a file
- downloaded a file

### CODE REVIEW

File: statements/downloaded_file.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+         "name": {
+            "en": "Test Course"
+         },
+         "extensions": {
+            "https://w3id.org/xapi/acrossx/extensions/type": "application/pdf"
+         }
+      }
+   }
+}
+```
```
Comment (quitterie-lcs) [inline statements/downloaded_file.md:35]: Profiles from this organization are documented with additional information about the `determining properties` and the `statements rules`.

````suggestion
```
## Determining properties

| Property  | Value         |
|----------------|-----------------|
| verb.id | Must be `https://w3id.org/xapi/netc/verbs/downloaded` |
| object.definition.type | Must be `http://activitystrea.ms/file` |

## Rules

- `context.extensions.https://w3id.org/xapi/acrossx/extensions/type`: <INCLUDED> or <RECOMMENDED>, give information about the value(s).
````

File: statements/downloaded_file.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      "account": {
+         "name": "john",
+         "homePage": "http://gaiax.org"
+      }
+   },
+   "verb": {
+      "id": "https://w3id.org/xapi/netc/verbs/downloaded"
+   },
+   "object": {
+      "objectType": "Activity",
```
Comment (quitterie-lcs) [inline statements/downloaded_file.md:22]:
```suggestion
```
This field is not mandatory. We try to minimize the statement content. The definition type is enough to give information that this is an activity and not an agent

File: statements/downloaded_file.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+## Description
+
+This event is emitted when a user has downloaded a file such as a pdf, doc, txt, ppt, xls, csv, etc.
+
+## Example
+
+```json
+{
+   "actor": {
+      "objectType": "Agent",
```
Comment (quitterie-lcs) [inline statements/downloaded_file.md:12]:
```suggestion
```

File: statements/downloaded_file.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Downloaded a file
+
+## Description
+
+This event is emitted when a user has downloaded a file such as a pdf, doc, txt, ppt, xls, csv, etc.
+
+## Example
+
+```json
+{
```
Comment (quitterie-lcs) [inline statements/downloaded_file.md:10]: I would recommend to add the timestamp (cf. [this discussion](https://github.com/gaia-x-dases/xapi-virtual-classroom/discussions/8) we had on the virtual classroom profile)

File: statements/uploaded_file.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+## Description
+
+This event is emitted when a user has uploaded a file such as a pdf, doc, txt, ppt, xls, csv, etc.
+
+## Example
+
+```json
+{
+   "actor": {
+      "objectType": "Agent",
```
Comment (quitterie-lcs) [inline statements/uploaded_file.md:12]:
```suggestion
```

File: statements/uploaded_file.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+      "account": {
+         "name": "john",
+         "homePage": "http://gaiax.org"
+      }
+   },
+   "verb": {
+      "id": "https://w3id.org/xapi/netc/verbs/uploaded"
+   },
+   "object": {
+      "objectType": "Activity",
```
Comment (quitterie-lcs) [inline statements/uploaded_file.md:22]:
```suggestion
```

File: statements/uploaded_file.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+         "name": {
+            "en": "Test Course"
+         },
+         "extensions": {
+            "https://w3id.org/xapi/acrossx/extensions/type": "application/pdf"
+         }
+      }
+   }
+}
+```
```
Comment (quitterie-lcs) [inline statements/uploaded_file.md:35]:
````suggestion
```
## Determining properties

| Property  | Value         |
|----------------|-----------------|
| verb.id | Must be `https://w3id.org/xapi/netc/verbs/uploaded` |
| object.definition.type | Must be `http://activitystrea.ms/file` |

## Rules

- `context.extensions.https://w3id.org/xapi/acrossx/extensions/type`: <INCLUDED> or <RECOMMENDED>, give information about the value(s).
````

File: statements/downloaded_file.md (hunk 1 above)
Comment (wilbrdt) [reply statements/downloaded_file.md:35]: Whoops you're right! Thanks!

File: profile/profile.jsonld
Comment (wilbrdt) [inline profile/profile.jsonld:1]: TODO: Update this file after updating profile on ADLnet

File: statements/uploaded_file.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+## Determining properties
+
+| Property  | Value         |
+|----------------|-----------------|
+| verb.id | Must be `https://w3id.org/xapi/netc/verbs/uploaded` |
+| object.definition.type | Must be `http://activitystrea.ms/file` |
+
+## Rules
+
+- `context.contextActivities.category`: INCLUDED, MUST contain an activity with the `https://w3id.org/xapi/lms` id.
```
Comment (wilbrdt) [inline statements/uploaded_file.md:57]: TODO: Recreate new `lms` profile under ADLnet

File: README.md [hunk 9]
```diff
@@ -1,14 +1,19 @@
-# xAPI profiles for Open Education
+# xAPI common profile for Open Education
```
Comment (quitterie-lcs) [inline README.md:1]:
```suggestion
# xAPI LMS profile for Open Education
```

File: README.md [hunk 10]
```diff
@@ -1,14 +1,19 @@
...
 
-This documentation intends to describe xAPI learning profile defined by the Gaia-X organisation.
+This documentation intends to describe a xAPI common learning profile defined by the Gaia-X organisation.
 
-## Common learning profiles
+## Common learning profile
```
Comment (quitterie-lcs) [inline README.md:5]:
```suggestion
## Statements
```

**Review by quitterie-lcs — APPROVED** [review]

File: README.md [hunk 11]
```diff
@@ -1,14 +1,19 @@
-# xAPI profiles for Open Education
+# xAPI common profile for Open Education
 
-This documentation intends to describe xAPI learning profile defined by the Gaia-X organisation.
+This documentation intends to describe a xAPI common learning profile defined by the Gaia-X organisation.
```
Comment (quitterie-lcs) [inline README.md:3]:
```suggestion
This documentation intends to describe LMS profile defined by the Gaia-X organisation.
```

**Review by quitterie-lcs — APPROVED** [review]

**Pull request merged**